website/docs/r/capacity_reservation_group.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "Compute"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_capacity_reservation_group"
+description: |-
+  Manages a Compute.
```

Comment:
It's not a `Compute`, it's a `Capacity Reservation Group`, change them all in this doc.